### ISSUE TITLE
[6.x]: alpha.14 - alt text field cannot save a blank value

### ISSUE THREAD

**Issue by joepagan** [craftcms/cms#19310]
### Description

If an asset has an alt text value saved against it, attempting to delete the value and re-save returns the prior value.

### Steps to reproduce

1. Visit asset with an alt field value
2. Delete the field value
3. Resave
4. Upon the page refreshing observe the prior value

### Expected behavior

A blank value to be saved against the asset

### Actual behavior

Attempted to save a blank value will refresh the page and the prior value of the alt field is shown as if nothing happened.

### Craft CMS version

6.0.0-alpha.14

### PHP version

8.5

### Operating system and version

ddev: laravel PHP 8.5, Server: nginx-fpm, 

### Database type and version

mysql8.4

### Installed plugins and versions

- none

**Comment by brandonkelly** [comment]
Thanks for reporting that! Craft 6.0.0-alpha.15 is out with a fix.
